Source/Core/Core/FifoPlayer/FifoRecordAnalyzer.cpp

```diff
@@ -227,78 +228,3 @@ void FifoRecordAnalyzer::WriteVertexArray(int arrayIndex, u8 *vertexData, int ve
 
 	FifoRecorder::GetInstance().WriteMemory(arrayStart, arraySize, MemoryUpdate::VERTEX_STREAM);
 }
-
-void FifoRecordAnalyzer::ProcessTexMaps()
```

Comment:
Do you think we should merge this PR as it is, or also wait for the next commits which inlines the functions above?